apps/expo/src/app/activity.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Text, View } from "react-native";
+import { AntDesign } from "@expo/vector-icons";
```

Comment:
for now you can just use placeholders like "?" for icons, since we will be updating these components with the design system